uaaextras/templates/email/body.html

```diff
@@ -22,7 +22,13 @@
             {{verification_url}}
             </li>
             <li>
-	    <b>Read the documentation</b> - After you register and log in, review the <a href="https://cloud.gov/docs/getting-started/accounts/#use-your-account-responsibly">acceptable uses and rules of behavior</a>. Then <a href="https://cloud.gov/docs/getting-started/setup/">set up your access and get started</a>.
+	          <b>Read the documentation</b> - After you register and log in, review the <a href="https://cloud.gov/docs/getting-started/accounts/#use-your-account-responsibly">acceptable uses and rules of behavior</a>.
+            </li>
+            <li>
+            If you signed up for a trial account (a sandbox), <strong>make sure you understand the <a href="https://cloud.gov/overview/pricing/free-limited-sandbox/#to-keep-in-mind-before-you-try-your-sandbox"><strong>sandbox policies and restrictions</a></strong>. Note that sandbox content is wiped regularly. If this isn’t you, don’t worry about this and skip to ...
```

Comment:
A sandbox is a free space rather than a free account; people with sandbox access might not have explicitly signed up for sandbox access - people automatically get a sandbox space when they sign into cloud.gov with a federal email address. (For example, a person might have invited a federal employee coworker to collaborate on a paid org, and they would also get a sandbox space when they sign in.) This text also may be confusing because sandbox content is not yet wiped regularly.